src/network/core/tcp_admin.h

```diff
@@ -347,7 +347,7 @@ class NetworkAdminSocketHandler : public NetworkTCPSocketHandler {
 	 * string  Name of the company.
 	 * string  Name of the companies manager.
 	 * uint8_t   Main company colour.
-	 * bool    Company is password protected.
+	 * bool    Company is password protected (always false since 15 due to removal of passwords).
```

Comment:
Maybe it would be better to make it always `true`, to make software using admin port think all companies are now protected? (as they, are just not with a password)

Reply:
That a good reason to set it to `true`.
Though it is not protected by a password, which is what the comment says.

I don't care. Anyone else with an opinion?

Reply:
So, change the comment to read: "Company is protected (always false ...)" :P

It is more that tooling that already exists isn't going: omg your server has unprotected companies!!!!, after this patch :) As that isn't true. And the intention of this field was ofc the "protected" part, not the "password" part :)